nextflow_schema.json

```diff
@@ -0,0 +1,368 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/epitopeprediction/master/nextflow_schema.json",
+    "title": "nf-core/epitopeprediction pipeline parameters",
+    "description": "A fully reproducible and state of the art epitope prediction pipeline.",
+    "type": "object",
+    "definitions": {
+        "input_output_options": {
+            "title": "Input/output options",
+            "type": "object",
+            "fa_icon": "fas fa-terminal",
+            "description": "Define where the pipeline should find input data and save output data.",
+            "required": [
+                "input",
```

Comment:
Not sure, but does this not cause a problem when the user would want to use "peptides" or "proteins" as input?

Reply:
Good point. I'm wondering if we should switch to a single input parameter `input` and flags to specify if it is peptide or protein input (`--peptides`, `--proteins`). What do you think?

Reply:
I guess this would also make sense to keep consistently for all nf-core pipelines. At least `eager` does it like this too.

Reply:
(...as you suggested)

Reply:
I will change that in a separate PR.